### PR TITLE
MHV-55167 shrinks minimum size for message action buttons.

### PR DESCRIPTION
**Pull request by KolbyVA**
## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

## Summary
Shrinks the minimum size of the message action buttons so that they do not go over the edge of the screen for certain screen sizes.

## Related issue(s)
https://jira.devops.va.gov/browse/MHV-55167
> Trash button is not visible (mobile issue)-See attached image
> 
> Trash button in conversation details is shown partially off the screen--Antonio/UCD
> 
> User Story: As a Secure Messaging User, I want to see the trash button when using a mobile device

## Testing done
Validated by UCD team, all unit tests passing locally

## Screenshots
Before:
image

After:
image

## What areas of the site does it impact?
mhv-secure-messaging

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/mhv-secure-messaging/components/MessageActionButtons/TrashButton.jsx [hunk 1]
```diff
@@ -69,12 +69,7 @@ const TrashButton = props => {
           <div className="vads-u-margin-right--0p5">
             <va-icon icon="delete" aria-hidden />
           </div>
-          <span
-            className="message-action-button-text"
-            data-testid="trash-button-text"
```
Comment (vsaleem) [inline src/applications/mhv-secure-messaging/components/MessageActionButtons/TrashButton.jsx:74]: Do we need to keep this id for tests?

**Review by vsaleem — DISMISSED** [review]

File: src/applications/mhv-secure-messaging/components/MessageActionButtons/TrashButton.jsx [hunk 2]
```diff
@@ -59,8 +59,8 @@ const TrashButton = props => {
         <button
           id="trash-button"
           type="button"
-          className="usa-button-secondary small-screen:vads-u-flex--3 vads-u-display--flex vads-u-flex-direction--row vads-u-justify-content--center vads-u-align-items--center"
-          style={{ minWidth: '100px' }}
+          className="usa-button-secondary small-screen:vads-u-flex--3 vads-u-display--flex vads-u-flex-direction--row vads-u-justify-content--center vads-u-align-items--center vads-u-padding-x--2"
+          style={{ minWidth: '90px' }}
```
Comment (oleksii-morgun) [inline src/applications/mhv-secure-messaging/components/MessageActionButtons/TrashButton.jsx:63]: appears this is a repeating line of code across 3 buttons. Would it make more sense to assign a dedicated class ?

**Review by alex-morgun-1 — DISMISSED** [review]
Approving with pending proposal to assign class for width attribute instead

**Review by vsaleem — APPROVED** [review]

**Review by alex-morgun-1 — APPROVED** [review]
👍🏻

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request merged**